rbi/core/thread.rbi

```diff
@@ -1613,38 +1613,16 @@ end
 # of how a [`SizedQueue`](https://docs.ruby-lang.org/en/2.7.0/SizedQueue.html)
 # works.
 class Thread::SizedQueue < Thread::Queue
```

Comment:
I think we want to add the `push`, `<<` and `enq` declarations here, because they 2 have extra params that `Queue` versions don't have: https://ruby-doc.org/3.3.3/Thread/SizedQueue.html#method-i-push

Reply:
Oops, good spot! Fixed now.